Test failure: TestRun/Without_ptr:_only_non-ptr_methods

I'm working on packaging this library for [Debian](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1059471), and have encountered a test failure when building.

Versions:
* go 1.21.5 linux/amd64
* github.com/maxatome/go-testdeep v1.13.0
* github.com/davecgh/go-spew v1.1.1

Here is what I hope is the relevant part of the test output:

```
=== RUN   TestRun/Without_ptr:_only_non-ptr_methods
    suite_test.go:181: Run(): several methods are not accessible as suite is not a pointer but tdsuite_test.FullNoPtr: PostTest, Test2
=== RUN   TestRun/Without_ptr:_only_non-ptr_methods/Test1
=== RUN   TestRun/Without_ptr:_only_non-ptr_methods/Test3
=== NAME  TestRun/Without_ptr:_only_non-ptr_methods
    suite_test.go:210: ^[[1;33mFailed test^[[0m
        ^[[1;36mDATA.output: does not contain^[[0m
        ^[[1;31m             got: ^[[0;31m([]uint8) (len=140 cap=160) {^[[0m
                          ^[[0;31m 00000000  20 20 20 20 2d 2d 2d 20  50 41 53 53 3a 20 54 65  |    --- PASS: Te|^[[0m
                          ^[[0;31m 00000010  73 74 52 75 6e 2f 57 69  74 68 6f 75 74 5f 70 74  |stRun/Without_pt|^[[0m
                          ^[[0;31m 00000020  72 3a 5f 6f 6e 6c 79 5f  6e 6f 6e 2d 70 74 72 5f  |r:_only_non-ptr_|^[[0m
                          ^[[0;31m 00000030  6d 65 74 68 6f 64 73 2f  54 65 73 74 31 20 28 30  |methods/Test1 (0|^[[0m
                          ^[[0;31m 00000040  2e 30 30 73 29 0a 20 20  20 20 2d 2d 2d 20 50 41  |.00s).    --- PA|^[[0m
                          ^[[0;31m 00000050  53 53 3a 20 54 65 73 74  52 75 6e 2f 57 69 74 68  |SS: TestRun/With|^[[0m
                          ^[[0;31m 00000060  6f 75 74 5f 70 74 72 3a  5f 6f 6e 6c 79 5f 6e 6f  |out_ptr:_only_no|^[[0m
                          ^[[0;31m 00000070  6e 2d 70 74 72 5f 6d 65  74 68 6f 64 73 2f 54 65  |n-ptr_methods/Te|^[[0m
                          ^[[0;31m 00000080  73 74 33 20 28 30 2e 30  30 73 29 0a              |st3 (0.00s).|^[[0m
                          ^[[0;31m}^[[0m
        ^[[1;32m        expected: ^[[0;32mContains("Run(): several methods are not accessible as suite is not a pointer but tdsuite_test.FullNoPtr: PostTest, Test2")^[[0m
        [under operator Contains at suite_test.go:211]
        This is how we got here:
                TestRun.func3() helpers/tdsuite/suite_test.go:210

[snip]

    --- FAIL: TestRun/Without_ptr:_only_non-ptr_methods (0.00s)
        --- PASS: TestRun/Without_ptr:_only_non-ptr_methods/Test1 (0.00s)
        --- PASS: TestRun/Without_ptr:_only_non-ptr_methods/Test3 (0.00s)
```

--- a/helpers/tdsuite/suite_test.go
+++ b/helpers/tdsuite/suite_test.go
@@ -178,7 +178,8 @@ func TestRun(t *testing.T) {
 	t.Run("Without ptr: only non-ptr methods", func(t *testing.T) {
 		defer traceFullNoPtr.clean()
 		suite := FullNoPtr{}
-		td.CmpTrue(t, tdsuite.Run(t, suite)) // non-ptr
+		tb := test.NewTestingTB("TestWithoutPtr")
+		td.CmpTrue(t, tdsuite.Run(tb, suite)) // non-ptr
 		ok := td.Cmp(t, traceFullNoPtr.calls, []string{
 			"Setup",
 			/**/ "PreTest+Test1",
@@ -207,8 +208,7 @@ func TestRun(t *testing.T) {
 			}
 		}
 		// Yes it is a bit ugly
-		td.Cmp(t, t, td.Smuggle("output",
-			td.Contains("Run(): several methods are not accessible as suite is not a pointer but tdsuite_test.FullNoPtr: PostTest, Test2")))
+		td.CmpEmpty(t, tb.ContainsMessages("Run(): several methods are not accessible as suite is not a pointer but tdsuite_test.FullNoPtr: PostTest, Test2"))
 	})
 
 	t.Run("With ptr: all ptr & non-ptr methods", func(t *testing.T) {

